### DISCUSSION TITLE
Window capture in OBS

### DISCUSSION THREAD

**Discussion by zeerooth** [leftwm/leftwm#1048] [Q&A]
Hello, I currently have a problem where I can't capture a window in OBS with xcomposite if it's in another desktop (I guess it's due to the fact that it's not being rendered at all)
Is there any way to work around that? Like forcing all desktop to be rendered at all times for example? I tried to group all my applications that I want captured in one workspace, along with OBS, but unfortunately whenever I switch they all automatically disappear, which is a pain for my setup...

**Comment by VuiMuich** [comment]
Kind of a workaround would be to make the windows you want to grab `Floating` and `Sticky` but this comes with its own downsides ofc.

I agree, that it would be nice to enforce rendering of a window despite it being actually visible. Just out of curiosity, do you have any reference on another WM that is handling this like your suggestion? I mean basically it is safe to consider hidden windows not to become rendered as correct behaviour...
